examples/rectangular_reinforced_cross_section.py

```diff
@@ -0,0 +1,59 @@
+"""Reinforced concrete cross-section example."""
+
+from blueprints.materials.concrete import ConcreteMaterial, ConcreteStrengthClass
+from blueprints.materials.reinforcement_steel import ReinforcementSteelMaterial, ReinforcementSteelQuality
+from blueprints.structural_sections.concrete.rebar import Rebar
+from blueprints.structural_sections.concrete.reinforced_concrete_sections.rectangular import RectangularReinforcedCrossSection
+
+# Define a concrete material
+concrete = ConcreteMaterial(concrete_class=ConcreteStrengthClass.C35_45)
+
+# Define a reinforcement steel material
+steel = ReinforcementSteelMaterial(steel_quality=ReinforcementSteelQuality.B500B)
+
+# Define a rectangular reinforced cross-section
+cs = RectangularReinforcedCrossSection(
+    width=1000,
+    height=800,
+    covers=(45, 30, 35, 50),  # upper, right, lower, left
+    concrete_material=concrete,
+)
+
+# Add reinforcement to the cross-section
+cs.add_longitudinal_reinforcement_by_quantity(
+    n=5,
+    diameter=14,
+    edge="upper",
+    material=steel,
+)
+cs.add_longitudinal_reinforcement_by_quantity(
```

Comment:
Can we add some examples on placing rebar by spacing instead of number? Think its good to showcase both options

Reply:
Done :-)